src/Compilers/CSharp/Portable/Syntax/InternalSyntax/SyntaxToken.cs

```diff
@@ -285,9 +286,9 @@ public virtual object Value
                 switch (this.Kind)
                 {
                     case SyntaxKind.TrueKeyword:
-                        return true;
+                        return Boxes.BoxedTrue;
                     case SyntaxKind.FalseKeyword:
-                        return false;
+                        return Boxes.BoxedFalse;
```

Comment:
>Boxes.BoxedFalse [](start = 31, length = 16)

Is similar change applicable to VB?

Reply:
@AlekseyTs VB is already using Boxes.

Reply:
VB already used `Boxes`:

https://github.com/dotnet/roslyn/blob/478adbc6624774e081b122acbd2d8787741bbbbf/src/Compilers/VisualBasic/Portable/Syntax/InternalSyntax/SyntaxToken.vb#L534-L537